app/src/main/java/com/poupa/vinylmusicplayer/ui/fragments/player/card/CardPlayerFragment.java

```diff
@@ -178,6 +178,7 @@ public void onQueueChanged() {
 
     @Override
     public void onMediaStoreChanged() {
+        // TODO If a song is removed from the MediaStore, this is not reflected in the playing queue untill restart
```

Comment:
Is this to not forget the work that need to be done ?

Reply:
Yup, this is not a regression and goes beyond the scope of this PR (to fix the crash)

Reply:
👍